app/views/albums/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Albums#edit</h1>
+<p>Find me in app/views/albums/edit.html.erb</p>
```

Comment:
We should remove this line and the one above it, as we do not actually want to display this text on the page.